### PR TITLE
[MRG] File Manager Controls

### PR DESCRIPTION
**Pull request by SomeKitten**
Added a few controls so that a few more things can be done with only the keyboard.

### CODE REVIEW

File: MapEditor.py [hunk 1]
```diff
@@ -188,6 +194,23 @@ def _get_paths(self):
...
+    def return_decompile(self, event):
+        self._decompile()
+
+    def tab_switch_dst(self, event):
+        self.dst_tv.focus_set()
+        return 'break'
+
+    def tab_switch_src(self, event):
+        self.src_tv.focus_set()
+        return 'break'
```
Comment (monkeyman192) [inline MapEditor.py]: why return `'break'`?

File: MapEditor.py [hunk 2]
```diff
@@ -188,6 +194,23 @@ def _get_paths(self):
             title="Select a directory to place the patch.")
         self.settings.write(self.paths)
 
+    # region public functions
+
+    def return_edit(self, event):
```
Comment (monkeyman192) [inline MapEditor.py]: method name doesn't really make sense... same with `return_decompile`

File: MapEditor.py (hunk 1 above)
Comment (SomeKitten) [reply MapEditor.py]: Tab has some built-in functionality. [This Stackoverflow answer](https://stackoverflow.com/questions/4090683/overriding-default-tab-behaviour-in-python-tkinter) fixed it. That was my implementation of it.

File: MapEditor.py (hunk 2 above)
Comment (SomeKitten) [reply MapEditor.py]: I can update it if you want. I was going off of the fact that it is triggered via the `<Return>` keybind. Any suggestions?

File: MapEditor.py (hunk 1 above)
Comment (monkeyman192) [reply MapEditor.py]: huh... TIL

File: MapEditor.py (hunk 2 above)
Comment (monkeyman192) [reply MapEditor.py]: either something like `call_decompile`, or just have it a public `decompile` function which then calls `_decompile`. That way if you want to tack extra things on you can add them to the `decompile` function

Comment (SomeKitten) [reply MapEditor.py]: Alright, I've updated their names to `edit` and `decompile`.

**Pull request merged**